### PR TITLE
Add eval metrics notebook

### PR DESCRIPTION
**Pull request by alex000kim**
Addresses @daavoo's TODO from another PR:
https://github.com/iterative/example-repos-dev/pull/161#discussion_r1088137376

### CODE REVIEW

**Comment by shcheklein** [comment]
@alex000kim does it close some ticket? could you please add a bit more details to understand the context better ... may be screenshot of screen recording would be also great, to help review this.

**Comment by alex000kim** [comment]
@shcheklein good point. I updated the description (not sure what you mean by a screen recording)

File: example-get-started-cv/code/notebooks/TrainSegModel.ipynb [hunk 1]
```diff
@@ -154,7 +172,8 @@
     "    live = Live(dir=os.path.join('results', 'train'), \n",
     "            report=\"md\", \n",
     "            save_dvc_exp=True)\n",
-    "    live.log_param(\"base_lr\", base_lr)\n",
+    "    live.summary[\"base_lr\"] =base_lr\n",
+    "    live.make_summary()\n",
```
Comment (daavoo) [inline example-get-started-cv/code/notebooks/TrainSegModel.ipynb:176]: @alex000kim any reason to not make this a parameter?

Comment (alex000kim) [reply example-get-started-cv/code/notebooks/TrainSegModel.ipynb:176]: Do you mean via `.log_param()`? 
I can update that, but for this one, I simply followed the docs examples for fastai:
https://dvc.org/doc/dvclive/api-reference/ml-frameworks/fastai#examples 
image

**Pull request merged**